### PR TITLE
Fix links to Microsoft CQRS and ES reference

### PR DESCRIPTION
**Pull request by octonato**
Related to #29594 

Here are two other broken rendering due to the same escape issue.

* https://doc.akka.io/docs/akka/current/persistence-query.html#performance-and-denormalization
* https://doc.akka.io/docs/akka/current/typed/persistence.html#event-sourcing-concepts

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: akka-docs/src/main/paradox/persistence-query.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Java
 
 ## Performance and denormalization
 
-When building systems using @ref:[Event sourcing](typed/persistence.md#event-sourcing-concepts) and CQRS ([Command & Query Responsibility Segregation](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj554200(v=pandp.10)?redirectedfrom=MSDN)) techniques
+When building systems using @ref:[Event sourcing](typed/persistence.md#event-sourcing-concepts) and CQRS ([Command & Query Responsibility Segregation](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj554200\(v=pandp.10\)?redirectedfrom=MSDN)) techniques
```
Comment (patriknw) [inline akka-docs/src/main/paradox/persistence-query.md:202]: I'm confused. How many of these links do we have? Isn't the right approach in https://github.com/akka/akka/pull/29576/files ?

Comment (renatocaval) [reply akka-docs/src/main/paradox/persistence-query.md:202]: The change in #29576 doesn't seem to work as expected. See https://doc.akka.io/docs/akka/current/typed/persistence.html#event-sourcing-concepts

File: akka-docs/src/main/paradox/typed/persistence.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ provides tools to facilitate in building GDPR capable systems.
 
 ### Event sourcing concepts
 
-See an [introduction to EventSourcing](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj591559%28v=pandp.10%29) at MSDN.
+See an [introduction to EventSourcing](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj591559\(v=pandp.10\)?redirectedfrom=MSDN) at MSDN.
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/typed/persistence.md:52]: FYI, this fix the change done in #29576

File: akka-docs/src/main/paradox/persistence-query.md (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/persistence-query.md:202]: but I think that the change in 29576 was merged after 2.6.9 (current)
it looks good in https://doc.akka.io/docs/akka/snapshot/typed/persistence.html#event-sourcing-concepts

@ignasi35 ?

Comment (renatocaval) [reply akka-docs/src/main/paradox/persistence-query.md:202]: Ah ok. Fine by me, but than in that case better to align. Either use `\` or `%28`

**Review by ignasi35 — CHANGES_REQUESTED** [review]
+1 to using `%`-encoding whenever possible (instead of custom escape syntaxes).

**Comment by renatocaval** [comment]
Did the requested changes. Using % escapes. 

I also removed a trailing `redirectedfrom=MSDN`. Not clear what that is, but has no impact on the links.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by octonato** [comment]
@ignasi35, can you do another review round?

**Review by ignasi35 — APPROVED** [review]

**Pull request merged**